### ISSUE TITLE
Invalid access token when trying to switch Dropbox accounts

### ISSUE THREAD

**Issue by jensVanbever** [karthik/rdrop2#161]
Hi,

So I made a script that succesfully pushes to and pulls from my dropbox account but now when I want to use my script from a more standard dropbox account of my company instead of my own personal one I can't get the new account to link to rdrop2.

With the first account everything went smooth, after the first drop_auth() my browser popped up and it was linked but now I can't get it to open, I've deleted the token file, reinstalled rdrop2, deleted the third party link on my personal dropbox account and logged in to the new account but I still can't get the window to pop up to connect. 

This is some code to show the error, I hope this is enough to give you an ideal of what I'm trying to explain:

library(rdrop2)
> drop_auth()
> drop_acc() %>% data.frame()

            error_summary                 .tag
1 invalid_access_token/.. invalid_access_token
> 
this is in R version 3.5

Thanks in advance if anyone knows what the (probably silly) problem is!

**Comment by ClaytonJY** [comment]
@jensVanbever did you manage to resolve this?

**Comment by jensVanbever** [comment]
yes I did, I installed the package "httpuv" and then ran the drop_auth(new_user = TRUE) which succeeded.
